lib/src/command/get.dart

```diff
@@ -27,8 +27,7 @@ class GetCommand extends PubCommand {
         help: "Report what dependencies would change but don't change any.");
 
     argParser.addFlag('precompile',
-        defaultsTo: true,
-        help: "Precompile executables and transformed dependencies.");
+        defaultsTo: true, help: "Precompile executables.");
```

Comment:
I think this would be fine - it might also be worthwhile to be a little more precise?

"Precompile executables in immediate dependencies."